lib/pipefy_message/providers/aws_client/sqs_broker.rb

```diff
@@ -44,11 +44,12 @@ def poller
           @poller.poll({ wait_time_seconds: @config[:wait_time_seconds],
                          message_attribute_names: ["All"], attribute_names: ["All"] }) do |received_message|
             payload = JSON.parse(received_message.body)
-            metadata = received_message.message_attributes.merge(received_message.attributes)
+            original_metadata = received_message.message_attributes.merge(received_message.attributes)
 
-            context = metadata["context"]
-            correlation_id = metadata["correlationId"]
-            event_id = metadata["eventId"]
+            metadata = transform_metadata(original_metadata)
+            context = metadata[:context]
```

Comment:
**suggestion:** I'm not sure whether I can commit to this branch, as you forked the gem, but I think we also need to change this in `lib/pipefy_message/consumer.rb` now (lines 76-78 inside `process_message`) - because we're `yield`ing from the SQS broker to the consumer abstraction, and now our `metadata` keys are symbols rather than strings.

I'm going to approve the MR once you change that :blush: